packages/@divvi/mobile/src/send/usePrepareSendTransactions.ts

```diff
@@ -54,18 +56,39 @@ export async function prepareSendTransactionsCallback({
 /**
  * Hook to prepare transactions for sending crypto.
  */
-export function usePrepareSendTransactions() {
-  const prepareTransactions = useAsyncCallback(prepareSendTransactionsCallback, {
-    onError: (error) => {
-      Logger.error(TAG, `prepareTransactionsOutput: ${error}`)
+
+export type UsePrepareSendTransactions = {
+  prepareTransactionsResult: PreparedTransactionsResult | undefined
+  refreshPreparedTransactions: (
+    props: PrepareSendTransactionsCallbackProps
+  ) => Promise<PreparedTransactionsResult | undefined>
+  clearPreparedTransactions: () => void
+  prepareTransactionError: Error | undefined
+  prepareTransactionLoading: boolean
+}
+
+export function usePrepareSendTransactions(
+  existingPrepareTranscation?: UsePrepareSendTransactions
```

Comment:
i'm imagining how we'd document how to use this variable and it's pretty difficult to explain, and given the niche use case i think it's better not to expose this to builders? wdyt?

Reply:
oh actually this is my bad, i misunderstood this as the exported `prepareTransactions` that we expose to builders. ignore this comment